python/test/t_Sample_operators.expout

```diff
@@ -12,6 +12,10 @@ sample*2:     [ a0 a1 ]
 0 : [ 60 80 ]
 1 : [ 62 82 ]
 2 : [ 64 84 ]
+2*sample:     [ a0 a1 ]
+0 : [ 60 80 ]
+1 : [ 62 82 ]
+2 : [ 64 84 ]
 sample/2:     [ a0   a1   ]
```

Comment:
@jschueller  f9796be should be in a new PR ?
Btw seems a bug here no ? 2*Sample == Sample ???